src/main/java/org/jenkinsci/plugins/github_branch_source/GitHubSCMSource.java

```diff
@@ -981,19 +982,22 @@ public GHPermissionType fetch(String username) throws IOException, InterruptedEx
                                             public SCMRevision create(@NonNull PullRequestSCMHead head,
                                                                       @Nullable Void ignored)
                                                     throws IOException, InterruptedException {
+                                                String baseHash = pr.getBase().getSha();
```

Comment:
This is not necessarily a meaningful value, as noted [here](https://issues.jenkins-ci.org/browse/JENKINS-33161?focusedCommentId=261620&page=com.atlassian.jira.plugin.system.issuetabpanels%3Acomment-tabpanel#comment-261620). It is in particular not necessarily the current head commit of the base branch (what is being calculated in the deleted code).

Reply:
This is not a change from the existing behavior - if you read the code, all I did was move the assignments/return statements around.  If you like, I can put them back so they don't show up in the diff.